### PR TITLE
#2 Developer guide (part 2, Use case and product survey)

### PR DESCRIPTION
**Pull request by Melvin-Tan**

### CODE REVIEW

**Comment by Melvin-Tan** [comment]
#2

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -357,30 +357,106 @@ Priority | As a ... | I want to ... | So that I can...
...
 
-3a. The given index is invalid
+#### Use case 4: Add alias command
 
-> 3a1. AddressBook shows an error message <br>
-  Use case resumes at step 2
+**MSS**
+
+1. Actor requests to `alias` a command in the input box with a `new alias name`.
+2. System finds the command and alias it.
```
Comment (louislai) [inline docs/DeveloperGuide.md]: I am unconvinced on this point. This is different from how alias works in other mature system (screen shot attached)

The main intention of alias is just to alias a long `command` into a short one. 
As long as the alias can then be used to give the same output as the actual command, the intention is fulfilled

Comment (louislai) [reply docs/DeveloperGuide.md]: screen shot 2017-02-27 at 20 38 08

**Comment by louislai** [comment]
need to merge this soon. pls handle all the comments

**Review by louislai — APPROVED** [review]

**Pull request merged**